Content.Server/Body/Components/MetabolizerComponent.cs

```diff
@@ -54,14 +54,13 @@ public sealed partial class MetabolizerComponent : Component
         /// </summary>
         [DataField]
         public bool RemoveEmpty = false;
-
         /// <summary>
-        ///     How many reagents can this metabolizer process at once?
+        ///     How many poisons can this metabolizer process at once?
         ///     Used to nerf 'stacked poisons' where having 5+ different poisons in a syringe, even at low
         ///     quantity, would be muuuuch better than just one poison acting.
         /// </summary>
-        [DataField("maxReagents")]
-        public int MaxReagentsProcessable = 3;
+        [DataField("maxPoisons")]
```

Comment:
```suggestion
        [DataField("maxReagents")]
```
This is why renaming DataFields is not a good idea.

![image](https://github.com/user-attachments/assets/e3d3403a-fe3f-4b81-941a-d9651e95662c)

Reply:
> This is why renaming DataFields is not a good idea.
> 
> ![image](https://private-user-images.githubusercontent.com/16548818/406680023-e3d3403a-fe3f-4b81-941a-d9651e95662c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc4Mjk1NjEsIm5iZiI6MTczNzgyOTI2MSwicGF0aCI6Ii8xNjU0ODgxOC80MDY2ODAwMjMtZTNkMzQwM2EtZmUzZi00YjgxLTk0MWEtZDk2NTFlOTU2NjJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTI1VDE4MjEwMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJkZTBlNzM4MWUyYmRjODFiMDIxZDE4YmIxOTZlMDgyOTEwYzE1ZjM1ZWRmZjVlOGIwZmYzODg5NDVjNTI4YTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.hVjTLR7xdk9l0OuLhwSpXdR20N21HOxon3z5iBoHVPI)

I think looking at that gave me a stroke